src/frontend/app/shared/components/list/list.component.scss

```diff
@@ -159,6 +159,10 @@
   animation-play-state: running;
 }
 
+.multi-action {
```

Comment:
Think this should be nuked, there's no `multi-action` class, and changing to `multi-actions` results in a loose vertical line